core/src/main/scala/tech/beshu/ror/accesscontrol/factory/GlobalSettings.scala

```diff
@@ -16,14 +16,13 @@
  */
 package tech.beshu.ror.accesscontrol.factory
 
-import tech.beshu.ror.accesscontrol.domain.{RorAuditIndexTemplate, RorConfigurationIndex}
+import tech.beshu.ror.accesscontrol.domain.RorConfigurationIndex
 
 final case class GlobalSettings(showBasicAuthPrompt: Boolean,
                                 forbiddenRequestMessage: String,
                                 flsEngine: GlobalSettings.FlsEngine,
                                 configurationIndex: RorConfigurationIndex,
-                                usernameCaseMapping: GlobalSettings.UsernameCaseMapping,
-                                indexAuditTemplate: Option[RorAuditIndexTemplate])
```

Comment:
Do we need `indexAuditTemplate` in global settings? It looks like it's unused, so I deleted it.